scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ServerJobAndTaskLogs.java

```diff
@@ -84,28 +84,26 @@ public static String getJobLog(JobId jobId, Set<TaskId> tasks) {
 
     }
 
-    public static void remove(JobId jobId, Set<TaskId> taskIds) {
-        removeLog(JobLogger.getJobLogFilename(jobId));
-        for (TaskId taskId : taskIds) {
-            removeLog(TaskLogger.getTaskLogFilename(taskId));
-        }
+    public static void remove(JobId jobId) {
+        removeFolderLog(jobId.value());
     }
 
-    private static void removeLog(String filename) {
-        for (String suffix : new String[] { "", ".1" }) {
-            removeFile(filename + suffix);
-        }
+    private static void removeFolderLog(String folderName) {
+        removeFile(folderName);
+
     }
 
     private static void removeFile(String path) {
         if (logsLocationIsSet()) {
             String logsLocation = getLogsLocation();
-            File f = new File(logsLocation, path);
-            if (f.exists()) {
-                logger.info("Removing file " + f.getAbsolutePath());
-                f.delete();
+            File logFolder = new File(logsLocation, path);
+            try {
+                org.apache.commons.io.FileUtils.deleteDirectory(logFolder);
```

Comment:
It is probably better to use deleteQuietly, throwing an exception here would be worse than the actual file deletion problem.